docs/src/guide/periodic_problems.jl

```diff
@@ -85,15 +85,15 @@
 # ```math
 #    -i∇ \left( e^{i k⋅x} f \right) 
 #    = -i\frac{d}{dx} \left( e^{i k⋅x} f \right)
-#    = k e^{i k⋅x} f -i∇ e^{i k⋅x} f = (-i∇ + k) e^{i k⋅x} f.
+#    = k e^{i k⋅x} f -i e^{i k⋅x} ∇f = e^{i k⋅x} (-i∇ + k) f.
```

Comment:
I usually prefer to keep (-i nabla) together (because it's the p operator and keeping it together avoids some sign mistakes)